cyclonedx/output/json.py

```diff
@@ -37,6 +38,11 @@ def generate(self, force_regeneration: bool = False) -> None:
         if self.generated and not force_regeneration:
             return
 
+        schema_uri: Optional[str] = self._get_schema_uri()
+        if not schema_uri:
```

Comment:
:heart: 
that is a great idea of a process
how to indicate that JSON serialization is supported for a specific spec,
and have qualified information about the used JSON schema
at the same time.